pytentiostat/main.py

```diff
@@ -3,19 +3,76 @@
 from pytentiostat.reporter import save_data_to_file
 from pytentiostat.tester import experiment
 from pytentiostat.routines import startup_routine, closing_routine
+from pytentiostat.plotter import plot_updater, plot_initializer
+import sys
+import time
+
+
+
+class BoardCom:
+    def __init__(self):
+        self.com, self.board, self.pin_a0, self.pin_a2, self.pin_d9 = startup_routine()
+
+    def configure_board(self):
+        self.com, self.board, self.pin_a0, self.pin_a2, self.pin_d9 = startup_routine()
+
 
 config_data = parse_config_file()
 
-# Initialize Experiment
-com, board, a0, a2, d9 = startup_routine()
+try:
+    # Initialize Experiment
+    board_instance = BoardCom()
+    # com, board, a0, a2, d9 = startup_routine()
+except KeyboardInterrupt:
+    sys.exit("Stopped. Make any changes and then restart the software.\nExiting...")
```

Comment:
Can this statement be more helpful.  Try and put your head inside a bewildered undergrad and write a statement that would help them.